workspace/utilities/admin-tools/admin-tools.css

```diff
@@ -5,8 +5,8 @@
 	background-color: #2f77eb;
 	color: white;
 	font-size: 14px;
-	font-family: 'Helevetica', 'Arial' sans-serif;
-	z-index: 99999;
+	font-family: 'Helevetica', 'Arial', sans-serif;
```

Comment:
Helvetica 😉

Reply:
yup @hotdoy this needs fixing, good find @fhamon